bundles/org.eclipse.e4.ui.model.workbench/model/UIElements.genmodel

```diff
@@ -339,7 +340,4 @@
       </nestedGenPackages>
     </nestedGenPackages>
   </genPackages>
-  <codeStyle>UnnecessaryAssignmentBeforeReturn</codeStyle>
```

Comment:
Why are these removed?  This will cause major style changes in the generate code.  Many of these options were developed purely because the e4 folks wanted these styles.

Reply:
That also explained most of the changes in the other model parts. I pushed the reverted version now